cmd/root.go

```diff
@@ -42,9 +45,23 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&leftDelimiter, "left-delimiter", "", "left delimiter")
 	rootCmd.PersistentFlags().StringVar(&rightDelimiter, "right-delimiter", "", "right delimiter")
 	rootCmd.PersistentFlags().BoolVar(&inPlace, "in-place", false, "template file without creating new one")
+	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "turn on debug logging")
+	rootCmd.PersistentFlags().Bool(consts.FailIfMissing, false, "will return an error if any variable is missing")
```

Comment:
For clarity sake I'd also make this a `BoolVar` instead of `Bool` - having additional var won't hurt us but it will make flags consistent

Reply:
done